stripe/res/layout/add_address_widget.xml

```diff
@@ -0,0 +1,113 @@
+<?xml version="1.0" encoding="utf-8"?>
+<merge xmlns:android="http://schemas.android.com/apk/res/android"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    >
+
+    <android.support.v7.widget.AppCompatSpinner
+        android:id="@+id/spinner_country_aaw"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        />
+
+    <android.support.design.widget.TextInputLayout
+        android:id="@+id/tl_name_aaw"
+        android:layout_width="match_parent"
```

Comment:
Did you want hints on any of these TextInputLayouts?

Reply:
![screen shot 2017-09-05 at 1 08 57 pm](https://user-images.githubusercontent.com/30061905/30081281-699d49c6-923b-11e7-9e7e-db2402be1a43.png)